bin/grant-server/main.go

```diff
@@ -68,7 +68,7 @@ func setupRouter(ctx context.Context, logger *zerolog.Logger) (context.Context,
 	r.Use(chiware.RealIP)
 
 	r.Use(chiware.Heartbeat("/"))
-	r.Use(chiware.Timeout(60 * time.Second))
+	r.Use(chiware.Timeout(10 * time.Second))
```

Comment:
Should this be set to the maximum timeout that we have? Say 15 instead of 10 seconds?

Reply:
Nevermind! misunderstood the relationship between chi and http.Server